packages/uikit/src/components/desktop/multi-send/MultiSendTable.tsx

```diff
@@ -0,0 +1,498 @@
+    display: flex;
+    gap: 0.5rem;
+    margin-right: auto;
+`;
+
+export const MultiSendTable: FC<{
+    className?: string;
+    list: MultiSendList;
+    onBack: () => void;
+}> = ({ className, list, onBack }) => {
+    const [asset, setAsset] = useState<TonAsset>(list.token);
+    const methods = useForm<MultiSendForm>({
+        defaultValues: list.form
+    });
+    const [confirmModalForm, setConfirmModalForm] = useState<MultiSendForm | undefined>();
+    const { isOpen, onClose, onOpen } = useDisclosure();
+
+    const { fields, append, remove } = useFieldArray({
+        control: methods.control,
+        name: 'rows'
+    });
+
+    const { mutate: updateList } = useMutateUserMultiSendList();
+
+    const onSubmit = (submitForm: MultiSendForm) => {
+        setConfirmModalForm(submitForm);
+        updateList({
+            form: {
+                rows: rowsValue
+            },
+            token: asset,
+            name: list.name,
+            id: list.id
+        });
+        onOpen();
+    };
+
+    const rowsValue = methods.watch('rows');
+
+    return (
+        <>
+            <AssetSelectWrapper>
+                <AssetSelect asset={asset} onAssetChange={setAsset} />
+            </AssetSelectWrapper>
+            <FormProvider {...methods}>
+                <AsyncValidatorsStateProvider>
+                    <TableFormWrapper
+                        onSubmit={methods.handleSubmit(onSubmit)}
+                        className={className}
+                    >
+                        <MultiSendTableGrid>
+                            {fields.map((item, index) => (
+                                <>
+                                    <FormRow key={item.id} index={index} asset={asset} />
+                                    <IconButtonStyled
+                                        type="button"
+                                        transparent
+                                        onClick={() => remove(index)}
+                                        hide={fields.length === 1}
+                                    >
+                                        <CloseIcon />
+                                    </IconButtonStyled>
+                                </>
+                            ))}
+                        </MultiSendTableGrid>
+                        <Button
+                            fitContent
+                            secondary
+                            type="button"
+                            onClick={() =>
+                                append({
+                                    receiver: undefined,
+                                    amount: undefined,
+                                    comment: ''
+                                })
+                            }
+                        >
+                            Add More
+                        </Button>
+                        <Spacer />
+                        <MultiSendFooter
```

Comment:
<img width="946" alt="Screenshot 2024-04-29 at 2 57 24 PM" src="https://github.com/tonkeeper/tonkeeper-web/assets/22742808/be70cee6-9d6b-4132-b489-1ed9277d07cd">

Could you please fix layout for buttons
thanks

Reply:
Done